plugins/labhub.py

```diff
@@ -108,6 +89,21 @@ def team_mapping(self):
             'maintainers': self.TEAMS[self.GH_ORG_NAME + ' maintainers'],
         }
 
+    @cmdfilter
+    def members_only(self, msg, cmd, args, dry_run):
+        user = msg.frm.nick
+        commands = ['assign_cmd', 'create_issue_cmd', 'invite_cmd', 'mark_cmd',
```

Comment:
ideally this is configurable, but that can be done after gsoc.

put it in constants for now.